examples/immersive_paris.html

```diff
@@ -171,6 +174,15 @@
                 // set camera to current panoramic
                 view.controls.setCameraToCurrentPosition();
                 view.notifyChange(view.camera.camera3D);
+
```

Comment:
Could you use `view.getLayerById('Building')` instead of `for loop`?

Reply:
yes, I've removed the loop. Finally I add the debug UI before,  when I add the layer.